LinearAlgebraForCAP/gap/LinearAlgebraForCAP.gi

```diff
@@ -812,6 +813,74 @@ InstallGlobalFunction( INSTALL_FUNCTIONS_FOR_MATRIX_CATEGORY,
                                   );
         
     end );
+
+    ## Operations related to the cohom-tensor adjunction
+
+    ##
+    AddCoDualOnObjects( category, { cat, obj } -> obj );
+
+    ##
+    AddCoDualOnMorphismsWithGivenCoDuals( category,
+      function( cat, codual_source, morphism, codual_range )
+        
+        return VectorSpaceMorphism( cat, codual_source,
+                                        TransposedMatrix( UnderlyingMatrix( morphism ) ),
+                                        codual_range );
```

Comment:
It would be nice if the noncompiled code is also tested. This is for several categories and should be done here as well. Maybe @zickgraf can help here.

Reply:
Should I adjust my tests from CategoryOfRows to work here, too? I don't think it's much work. 

Fabian probably once showed me how to use the noncompoiled code (or wrote an Email?) but I can't remember/find it. :(

Reply:
Testing everything twice, once with compiled and once without compiled code, is on my roadmap and in principle already works (simply applying `PackageJanitor` will introduce suitable test calls). However, there are still some drawbacks which I will have to work out. Can we merge this PR without testing the noncompiled code, or should I prioritize setting everything up properly?

PS: Sorry for the late reply, I got a bit side-tracked at the end of last week.

Reply:
I am OK with merging.